action.yml

```diff
@@ -23,9 +46,9 @@ inputs:
     required: false
     default: ''
   model:
-    description: 'OpenAI model to use'
+    description: 'Model to use (provider-specific, e.g., gpt-4-turbo-preview for OpenAI, gemini-pro for Google)'
```

Comment:
📝 Update the description to clarify that the model is provider-specific, which enhances understanding.